src/lib.rs

```diff
@@ -11,7 +11,7 @@ pub fn left_pad(s: &str, pad: u32) -> String
 /// pad a string to the left with ``pad`` ``padchar``
 pub fn left_pad_char(s: &str, pad: u32, padchar: char) -> String
 {
-    let mut out = String::new();
+    let mut out = String::with_capacity( s.as_bytes().len() + (pad as usize) );
```

Comment:
also the behaviour of the function was wrong and was fixed in 0.2.0, so you will have to fix that too.